<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar nueva categoría</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color : #FFF;
	margin : 0px;
	padding : 0px;
}

QFrame#content {
	background-color : transparent;
	border: 1px solid #DADDE1;
	border-radius: 5px;
	min-width : 450px;
	min-height : 350px;
}

QLabel {
	font : 77 12px &quot;Arial&quot;;
	color : green;
}

QLineEdit, QTextEdit, QComboBox, QSpinBox {
	background-color : transparent;
	border : 1px solid #DADDE1;
	border-radius : 5px;
	color : #616161;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
}

QPushButton {
	color : #FFFFFF;
	border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

#btn_file_explorer {
	background-color : #616161;
}

#btn_file_explorer:pressed {
	background-color : transparent;
	color : #616161;
	border: 1px solid #616161;
	border-radius: 5px;
}

#btn_accept_and_new, #btn_accept {
	background-color : #42B72A;
}

#btn_accept_and_new:pressed, #btn_accept:pressed {
	background-color : transparent;
	color : #42B72A;
	border: 1px solid #42B72A;
	border-radius: 5px;
}

#btn_cancel {
	background-color : red;
}

QPushButton#btn_cancel:pressed {
	background-color : transparent;
	color : red;
	border: 1px solid red;
	border-radius: 5px;
}

#lbl_image {
	min-width : 42px;
	max-width : 42px;
	min-height: 42px;
	max-height : 42px;
	border-radius : 5px;
    background-repeat : no-repeat;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="content">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_info">
        <property name="text">
         <string>* Rellene los campos para continuar con el registro.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="tfd_name">
        <property name="placeholderText">
         <string> * Nombre de la categoría</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="ted_description">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="placeholderText">
         <string> * Descripción</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_image">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets.qrc">:/images/assets/img/logo_python.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tfd_image">
           <property name="placeholderText">
            <string> Imagen del producto</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_file_explorer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Cargar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_info2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_cancel">
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_accept">
          <property name="text">
           <string>Agregar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_accept_and_new">
          <property name="text">
           <string>Agregar más</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
